pkg/reconcilermanager/controllers/reconciler_container_resources.go

```diff
@@ -55,7 +55,7 @@ func ReconcilerContainerResourceDefaults() map[string]v1beta1.ContainerResources
 		},
 		reconcilermanager.GCENodeAskpassSidecar: {
 			ContainerName: reconcilermanager.GCENodeAskpassSidecar,
-			CPURequest:    resource.MustParse("10m"),
+			CPURequest:    resource.MustParse("50m"),
```

Comment:
The problem this fixes is autopilot specific. On standard, the askpass container can boost. So I don't know if we want to increase the requirements for standard to fix autopilot. This is why I wrote https://github.com/GoogleContainerTools/kpt-config-sync/pull/872 to split apart the standard and autopilot resources.

/cc @janetkuo

Reply:
This will fix CI failures and unblock the upcoming 1.16.2 release next week.
I would suggest that we make the change and consolidate with #872 later.